mkalias/mkalias.c

```diff
@@ -114,14 +121,26 @@ int main (int argc, const char * argv[])
     return EX_NOINPUT;
   }
 
+  // check if the destination is an existing folder, if so create alias inside of destination folder
+  filename = basename(strdup(argv[optind]));
+  char newDest[strlen(destPath) + strlen(filename)];
+  if (UnixIsExistingFolder(/*destination*/(char *)destPath) == true)
+  {
+    strcpy(newDest, destPath);
+    strcat(newDest, "/");
+    strcat(newDest, (char *)filename);
+    destPath = newDest;
+    // printf("Destination is an existing folder, changing to new location: %s\n", (char *) destPath);
```

Comment:
What value does this commented out log statement add?

Reply:
not much probably. it should get stripped out when being compiled